### PR TITLE
Allow g:vimwiki_listsyms be larger than 5 elements (Issue #275)

### PR DESCRIPTION
**Pull request by Nudin**
Allows g:vimwiki_listsyms to be of arbitrary size, so that smoother transitions like '▁▂▃▄▅▆▇▉' or '○◔◑◕●✓' are possible.

### CODE REVIEW

**Comment by EinfachToll** [comment]
This is cool in principle, but there are a few more places in the source code which have to be adjusted. If you grep for `listsyms` in the source code you can easily fix it, so that the highlight works again.

**Comment by Nudin** [comment]
Thanks for the feedback. I've now hopefully adjusted all occurrences of `listsyms `. For the HTML-Version I so far translate back to the 5-step-system since I couldn't figure out a possibility to support arbitrary number of symbols there.

**Comment by Nudin** [comment]
Any further opinions to this Pull request?

**Comment by EinfachToll** [comment]
I forgot to hit the *Submit review* button. There are still some issues with your patch.

File: autoload/vimwiki/html.vim [hunk 1]
```diff
@@ -860,8 +860,12 @@ function! s:process_tag_list(line, lists) "{{{
     let chk = matchlist(a:line, a:rx_list)
     if !empty(chk) && len(chk[1]) > 0
       let completion = index(g:vimwiki_listsyms_list, chk[1])
-      if completion >= 0 && completion <=4
+      let n = len(g:vimwiki_listsyms_list)
+      if completion >= 0 && completion < n-1
+        let completion = float2nr(ceil(completion * 4.0 / (n-1) ))
```
Comment (Nudin) [inline autoload/vimwiki/html.vim]: Is fixed now.

File: autoload/vimwiki/html.vim [hunk 2]
```diff
@@ -861,11 +861,11 @@ function! s:process_tag_list(line, lists) "{{{
     if !empty(chk) && len(chk[1]) > 0
       let completion = index(g:vimwiki_listsyms_list, chk[1])
       let n = len(g:vimwiki_listsyms_list)
-      if completion >= 0 && completion < n-1
-        let completion = float2nr(ceil(completion * 4.0 / (n-1) ))
+      if completion == 0
+        let st_tag = '<li class="done0">'
+      elseif completion > 0 && completion < n
+        let completion = float2nr(round(completion / (n-1.0) * 3.0 + 0.5 ))
```
Comment (Nudin) [inline autoload/vimwiki/html.vim]: Here you can see the function:
http://www.wolframalpha.com/input/?i=plot+round(x*3%2B0.5)+for+x+in+%5B0,1%5D

**Pull request merged**